tests/units/Transvision/Utils.php

```diff
@@ -471,15 +473,19 @@ public function redirectToAPIDP()
     /**
      * @dataProvider redirectToAPIDP
      */
-    public function testRedirectToAPI($a, $b)
+    public function testRedirectToAPI($a, $b, $c)
     {
         $obj = new _Utils();
-        $_SERVER["REQUEST_URI"] = $a;
+        $_SERVER['REQUEST_URI'] = $a;
         $_SERVER['QUERY_STRING'] = isset(parse_url($a)['query'])
             ? parse_url($a)['query']
             : null;
+        $_SERVER['HTTP_HOST'] = 'http://' . parse_url($a)['host'];
```

Comment:
I don't think you should do that because you are adding even more transformation logic in your tests of globals, so you are testing something that is no longer exactly what the real function returns. Instead of rebuilding a full URL to match what we have in the data provider function, I think you should just remove the host from the strings in the data provider and only keep it as a set of REQUEST_URIs because this is what redirectToAPI() works from and returns.